### PR TITLE
Fix file exists

### PR DESCRIPTION
**Pull request by jeffeardley**
This pull request simply adds a check to see if the dataset already exists when running download_data.sh. It gives the user the opportunity to overwrite the existing file instead of erroring out.

### CODE REVIEW

File: download_data.py [hunk 1]
```diff
@@ -19,7 +20,9 @@ def download_data(
     """
     # Create folder to save this dataset's files in
     dataset_dir = Path(dataset_root_dir) / dataset_name
-    dataset_dir.mkdir(parents=True)
+    # Create the directory if it does not exist
```
Comment (nprisbrey) [inline download_data.py]: Would this comment be redundant?

File: download_data.py [hunk 2]
```diff
@@ -19,7 +20,9 @@ def download_data(
...
+    dataset_dir.mkdir(parents=True, exist_ok=True)
+
```
Comment (nprisbrey) [inline download_data.py:25]: I'd remove the extra carriage return.

**Review by nprisbrey — CHANGES_REQUESTED** [review]

File: download_data.py [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import datasets
+import shutil
```
Comment (nprisbrey) [inline download_data.py]: This should also be cleaned out not that it's not needed.

**Review by nprisbrey — CHANGES_REQUESTED** [review]

**Review by nprisbrey — APPROVED** [review]

**Review by KimballNJardine — APPROVED** [review]
This looks good.

**Pull request merged**